### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by TechStudent11**

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
@TechStudent11 What about the feature request template?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (KendallDoesCoding) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]:
```suggestion
labels: 'bug'
```

Add bug label to any issues created under the "Bug Report" issue template.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```
Comment (KendallDoesCoding) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]:
```suggestion
title: '[BUG REPORT]:- (Enter short, one sentence summary of bug here)'
```

Add title to Bug Report issue template.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```
Comment (KendallDoesCoding) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]:
```suggestion
name: 🐛 Bug Report
```

Update name of issue template.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+about: Create a report to help us improve
```
Comment (KendallDoesCoding) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a report to help us improve the project :)
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Comment by Kira272921** [comment]
Thanks for the lovely suggestions @KendallDoesCoding and also thanks a lot @TechStudent11 for creating a PR

**Comment by KendallDoesCoding** [comment]
> Thanks for the lovely suggestions @KendallDoesCoding and also thanks a lot @TechStudent11 for creating a PR

Welcome. I don't think https://github.com/Kira272921/vscode-themed-portfolio/pull/13/files/cbbb8e61c73b399a6e3c04724c987ed130868368#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5 got merged in for some reason.

**Comment by TechStudent11** [comment]
@KendallDoesCoding ah sorry I used GitHub's tool to create it. Looks like it wasn't effective. Thanks for noticing.

@Kira272921 No problem!

**Comment by KendallDoesCoding** [comment]
> @KendallDoesCoding ah sorry I used GitHub's tool to create it. Looks like it wasn't effective. Thanks for noticing.
> 
> @Kira272921 No problem!

No issues! It's weird that Github's tool doesn't automatically add the label as bug tho.

**Comment by Kira272921** [comment]
@all-contributors please add @TechStudent11 for doc

**Pull request merged**